jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -997,27 +1000,14 @@ YUI.add('deployment-flow', function() {
       @returns {Object} The React elements.
     */
     _generateDirectDeploy: function() {
-      const props = this.props;
-      const ddEntityId = props.ddData && props.ddData.id;
-      if (!ddEntityId) {
```

Comment:
By removing this we'll always be rendering the DD UI, which we don't want to do as it's the special case path. Modifying this will then require the addition of the tests below again.